Organic/Organic.php

```diff
@@ -740,7 +742,13 @@ public function syncPost( $post ) {
         $sponsorship = \apply_filters( 'organic_post_sponsorship', $post->post_sponsorship, $post->ID );
         $content = \apply_filters( 'organic_post_content', $post->post_content, $post->ID );
         $is_published = \apply_filters( 'organic_post_is_published', $post->post_is_published, $post->ID );
-        $published_date = \apply_filters( 'organic_post_publish_date', $post->post_date, $post->ID );
+
+        $post_meta_modified_date =  get_post_meta( $post->ID, STORY_TIMESTAMP_META_KEY, true );
```

Comment:
Organic WP plugin could be used not only by SWP but by any client with Wordpress installation. So, we shouldn't rely or depend on SWP internal logic.

A similar problem was discussed here: https://github.com/orgnc/wordpress-plugin/pull/31#discussion_r759560774

Instead of updating the plugin code, the correct `modified_date` should be passed from the SWP side through WP filters. Something like this: https://github.com/orgnc/solutions-wordpress/blob/master/src/web/app/themes/empire/app/filters.php#L2003